app/helpers/welcome_message_helper.rb

```diff
@@ -9,26 +9,29 @@ def random_welcome_message
   def welcome_messages
     [
       "Thank you! - Team CM",
+      "¡Gracias! - Equipo CM",
```

Comment:
It might be neat to have welcome_messages take a locale and spit out something related to it